release-notes/axon-framework-extensions.md

```diff
@@ -0,0 +1,24 @@
+# Axon Framework Extensions
```

Comment:
@smcvb ...we should provide a full list to bring it in alignment....can you point me to the list?

Reply:
There's not a single point where this can be found, other than the release note pages of all the extensions:

- https://github.com/AxonFramework/extension-amqp/releases
- https://github.com/AxonFramework/extension-cdi/releases
- https://github.com/AxonFramework/extension-jgroups/releases
- https://github.com/AxonFramework/extension-kafka/releases
- https://github.com/AxonFramework/extension-mongo/releases
- https://github.com/AxonFramework/extension-springcloud/releases
- https://github.com/AxonFramework/extension-tracing/releases

Reply:
Thanks @smcvb ...will add these on a separate PR...for now will merge this PR